src/renderers/shared/stack/reconciler/ReactOwner.js

```diff
@@ -7,12 +7,28 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactOwner
+ * @flow
  */
 
 'use strict';
 
 var invariant = require('invariant');
 
+import type { ReactInstance } from 'ReactInstanceType';
+
+/**
+ * @param {?object} object
+ * @return {boolean} True if `object` is a valid owner.
+ * @final
+ */
+function isValidOwner(object: any): bool {
```

Comment:
Don't worry about it since it makes no difference in this case - thanks for explaining. :)